src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -450,7 +450,7 @@
     }
   },
   "action-bar-height": {
-    "value": "48px",
+    "value": "56px",
```

Comment:
I think the intent for this is to be an alias that points to `component-height-400` instead of the pixel value directly, based on the changelog [here](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=20398-191676&t=OYhObFIvR6KJvF5e-4).